flavors/standard-EXASOL-8.0.0/FLAVOR_DESCRIPTION.md

```diff
@@ -0,0 +1,14 @@
+# Details for standard-EXASOL-8.0.0 flavor
+
+## Packages
+
+**Remarks:** The packages are grouped by there main usage and their type.
+
+- [UDFclient dependencies](flavor_base/udfclient_deps/packages/apt_get_packages)
+- [Language dependencies](flavor_base/language_deps/packages/apt_get_packages)
+- Flavor packages
+  - [Ubuntu packages](flavor_base/flavor_base_deps/packages/apt_get_packages)
+  - [Python3 pip packages](flavor_base/flavor_base_deps/packages/python3_pip_packages)
+- Customization
+  - [Ubuntu packages](flavor_customization/packages/apt_get_packages)
+  - [Python3 pip packages](flavor_customization/packages/python3_pip_packages)
```

Comment:
yes, because it's default and to be honest, there are no good other sources for python binaries. The PPA Deadsnakes can't give SLAs and Python 3.10 will be only included in Ubuntu 22.04 LTS and Python 3.6 is eol.